nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -150,7 +148,7 @@ def _remove_loading_dotdot(path):
 class FileTransferModule(CommandModule):
     """Command module with commands relevant to file transfer."""
 
-    def __init__(self, upload_dir: str, download_dir: str, upload_folder_cmd_name="upload_app"):
+    def __init__(self, upload_dir: str, download_dir: str, upload_folder_cmd_name="submit_jobs"):
```

Comment:
should this be submit_job not submit_jobs ? ( previous update_app), one job at time, not multiple jobs ( based on demo)

Reply:
Yes but this variable is not used so it has no effect. Will remove it when I get a chance.